app.js

```diff
@@ -1,16 +1,21 @@
 const express = require('express')
+require('dotenv').config()
+const config = require('./config')[process.env.NODE_ENV]
+
 const app = express()
 const http = require('http').createServer(app)
 
 app.set('view engine', 'ejs')
 app.use(express.static('public'))
 
 app.get('/', (req, res) => {
-  res.render('index', { page: 'index' })
+  res.render('index', { page: 'index', port: config.development.port })
```

Comment:
This was a mistake on my part, you do not need to pass port to render.
Please replace with:
`res.render('index', { page: 'index' })`